### DISCUSSION TITLE
[coapp] New Linux builds available with system ffmpeg support

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#47] [Announcements]
⚠️ this is for advanced users ⚠️

Thanks to @kuanyui [work](https://github.com/aclap-dev/vdhcoapp/pull/188), it is now possible install, on Linux, a version of the CoApp that relies on the system _ffmpeg_ (regular CoApp comes with its prebuilt _ffmpeg_).

To find the builds, just look at the _noffmpeg_ note here: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-Installation

**Comment by GrampaWildWilly** [comment]
Interesting . . . What is your stance on a Windows user substituting a version of ffmpeg obtained from ffmpeg.org into the CoApp directory?  Have you (that's plural, Paul + Michel) made any modifications to the source of ffmpeg?  Or is the ffmpeg packaged with the CoApp simply a build you guys did yourselves without any source code modifications?

By the way, some Windows users are EXTREMELY TECHNICAL with deep programming background.  Ditto for Mac users.  Being technical is not the exclusive purview of Linux users.

**Reply by paulrouget** [reply]
> Have you (that's plural, Paul + Michel) made any modifications to the source of ffmpeg? 

Yes. The changes are mostly about making ffmpeg less strict. But that system ffmpeg will do fine in most cases.

> By the way, some Windows users are EXTREMELY TECHNICAL with deep programming background. Ditto for Mac users. Being technical is not the exclusive purview of Linux users.

Sure. But the patch was for Linux.

**Reply by GrampaWildWilly** [reply]
Hmmmm.....  I like the idea of less strict.  I'm sure those changes have made things that were failing in VDH start to work.  But then that begs the question:  Aren't Linux users also going to lose the bug fixes for the cases that are cured by the less strict ffmpeg?

**Reply by paulrouget** [reply]
This is a special "noffmpeg" build for people who don't want ffmpeg. The default builds use builtin ffmpeg, not system.
